helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/ClusterContext.java

```diff
@@ -142,6 +146,10 @@ public float getEstimatedTopStateMaxUtilization() {
     return _estimatedTopStateMaxUtilization;
   }
 
+  public Map<String, Integer> getEstimateUtilizationMap() {
+    return _estimateUtilizationMap;
+  }
```

Comment:
I don't think we plan to change this map after its creation. Let's make this immutable or at least unmodifiable.